modules/cli/pages/index.adoc

```diff
@@ -54,9 +56,146 @@ $ mv skupper $HOME/bin
+* `internal` -  Use {skupper-name} authentication, see the `console-user` and `console-password` options.
+* `unsecured` - No authentication, anyone with the URL can view the console.
+
+
+`--console-user <username>`:: Username for the console user when authentication mode is set to `internal`.
+`--console-user <password>`:: Password for the console user when authentication mode is set to `internal`.
+--
+
+// Type: procedure
+[id="linking-sites"] 
+== Linking sites
+
+A {service-network} consists of {skupper-name} sites.
+This section describes how to link sites to form a {service-network}.
+
+Linking two sites requires a single initial directional connection. However:
+
+* Communication between the two sites is bidirectional, only the initial linking is directional.
+* The choice of direction for linking is typically determined by accessibility. For example, if you are linking an OpenShift Dedicated cluster with a CodeReady Containers cluster, you must link from the CodeReady Containers cluster to the OpenShift Dedicated cluster because that route is accessible. 
+
+.Procedure
+
+. Determine the direction of the link. If both clusters are publicly addressable, then the direction is not significant. If one of the clusters is addressable from the other cluster, perform step 2 below on the addressable cluster.
+
+. Generate a token on the cluster that you want to link to:
++
+--
+----
+$ skupper token create <filename> 
+----
+
+where `<filename>` is the name of a YAML file that is saved on your local filesystem.
+
+This file contains a key and the location of the site that created it.
+
+NOTE: Access to this file provides access to the {service-network}. Protect it appropriately.
+--
+
+. Use a token to join the {service-network}:
++
+--
+----
+$ skupper link create <filename> [-name <link-name>]
+----
+where where `<filename>` is the name of a YAML file generated from the `skupper token create` command and `<link-name>` is the name of the link.
+--
+
+. If you want to delete a link:
++
+----
+$ skupper link delete <link-name>
+----
+where `<link-name>` is the name of the link specified during creation.
+
+
+// Type: procedure
+[id="exposing-services"] 
+== Exposing services on the {service-network}
+
+After creating a {service-network}, services can communicate across that network.
+This section describes how services can be enabled for a {service-network}.
+
+.Procedure
+
+. Create a deployment, some pods, or a service in one of your sites, for example to create the backend service for the link:{getting-started-link}[tutorial]:
++
+----
+$ kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
+----
++
+This step is not {skupper-name}-specific, that is, this process is unchanged from standard processes for your cluster.
+
+
+. Expose the service on the  {service-network} using the `expose` option:
++
+--
+----
+$ skupper expose [deployment <name>|pods <selector>|service <name>]
```

Comment:
Suggest using the sequence:
$ skupper service create ...
$ skupper service bind ...

rather than the expose command (which combines service create and bind under the hood). This will give
the user a better sense of the concept of service definitions and bind targets

Reply:
I agree with Andy... and just for the records, the `expose` as well as the `bind` commands now accept `statefulset` as well.